Comments_Dump.sql

```diff
@@ -8,3 +8,11 @@ INSERT INTO Comments VALUES(2, 'Loving my new Pureboost Shoes!');
 INSERT INTO Comments VALUES(10, 'The phone came in good condition.');
 INSERT INTO Comments VALUES(2, 'Once bitten twice shy, wouldn''t want to try.');
 INSERT INTO Comments VALUES(3, 'Will it be the same quality if I got it from Royal Sporting House?');
+INSERT INTO Comments VALUES(, 'Bundle was missing 1 pair of socks :( It only came in 5 in 1.');
```

Comment:
Fill up the corresponding order_ids, liase with James on this since he worked on Order stuffs

Reply:
Will do so after james completes his part